inc/customizer.php

```diff
@@ -33,6 +33,52 @@ function twentynineteen_customize_register( $wp_customize ) {
 			)
 		);
 	}
+
+	// Add page primary color setting and control.
+	$wp_customize->add_setting( 'primary-color', array(
+		'default'           => '#0073aa',
+		'sanitize_callback' => 'sanitize_hex_color',
+		'transport'         => 'postMessage',
+	) );
+
+	$wp_customize->add_control( new WP_Customize_Color_Control( $wp_customize, 'primary-color', array(
```

Comment:
As was noted in the colors issue, it would be more appropriate to use a hue picker than the full color picker for this theme, similar to Twenty Seventeen. This is enabled, here, by adding the line:
`'mode'     => 'hue',`

The sanitize callback and default values should be updated to `absint` and the integer equivalent accordingly. All of the CSSS would then be revised to use HSL equivalent colors for the custom color  options. While this requires a minor tenchical effort (should take less than an hour), it eliminates the concerns regarding color contrast and makes accessibility much more achievable for end users.